jwt/src/main/java/me/uport/sdk/jwt/JWTTools.kt

```diff
@@ -257,33 +271,36 @@ class JWTTools(
 
         val signingInputBytes = token.substringBeforeLast('.').toByteArray(utf8)
 
-        val sigData = signatureBytes.decodeJose()
-
         val signatureIsValid = verificationMethod[header.alg]
-            ?.invoke(publicKeys, sigData, signingInputBytes)
+            ?.invoke(publicKeys, signatureBytes, signingInputBytes)
```

Comment:
making sure to pass along the raw signature bytes and leaving the decoding to the verifier method